frontend/src/router/index.ts

```diff
@@ -0,0 +1,59 @@
+import { createRouter, createWebHistory, RouteRecordRaw } from "vue-router";
+import Login from "../components/Login.vue";
+import CreateFlist from "../components/CreateFlist.vue";
+import Home from "../components/Home.vue";
+import UserFlist from "../components/UserFlist.vue";
+import FollowUp from "../components/FollowUp.vue";
+import PreviewFlist from "../components/PreviewFlist.vue";
```

Comment:
This would import all components even if we don't use them. Use lazy loading instead. https://router.vuejs.org/guide/advanced/lazy-loading.html